grpc/service_config/service_config.proto

```diff
@@ -328,6 +329,18 @@ message WeightedTargetLoadBalancingPolicyConfig {
   map<string, Target> targets = 1;
 }
 
+// Config for RLS LB policy.
+message RlsLoadBalancingPolicyConfig {
+  // Route lookup config.
```

Comment:
IMO it's better to not have a comment then to have a comment that doesn't provide any additional information.

Reply:
Done.